101-web-app-certificate-from-key-vault/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
This template assumes that a Web App already exists with a custom hostname assigned to it. I am not sure how changing placeholder parameter values would make it pass the automated PR test.